core/jobs_test.py

```diff
@@ -789,9 +789,23 @@ def test_each_one_off_class_is_subclass_of_base_job_manager(self):
         for klass in jobs_registry.ONE_OFF_JOB_MANAGERS:
             self.assertTrue(issubclass(klass, jobs.BaseJobManager))
 
+    def test_is_abstract_method_raises_exception_for_abstract_classes(self):
+        class TestMockAbstractClass(jobs.BaseJobManager):
+            """A sample Abstract Class."""
+            pass
+
+        mock_abstract_base_class = [TestMockAbstractClass]
```

Comment:
Technically this should be mock_abstract_base_classes since it's a list.

Reply:
Done.